doc/rados/configuration/common.rst

```diff
@@ -123,11 +123,11 @@ OSD host, run the following commands:
     ssh {osd-host}
     sudo mkdir /var/lib/ceph/osd/ceph-{osd-number}
 
-The ``osd_data`` path ought to lead to a mount point that has mounted on it a
-device that is distinct from the device that contains the operating system and
-the daemons. To use a device distinct from the device that contains the
-operating system and the daemons, prepare it for use with Ceph and mount it on
-the directory you just created by running the following commands:
+The ``osd_data`` path must lead to a mount point that has a device mounted on
+it that is not the device that contains the operating system and the
+daemons. To use a device other than the device that contains the operating
+system and the daemons, prepare it for use with Ceph and mount it on the
+directory you just created by running commands of the following form:
```

Comment:
The first sentence here reads as stilted.  Suggest

>The ``osd_data`` path must lead to a device that is not shared with the operating system.  To use such a device, prepare ...

That said, I *think* this is actually moot now that Filestore is finally deprecated, but it remains best practice for other reasons.

Reply:
I like your sentence much better than the one I wrote.

Accepted.